code/game/machinery/computer/cloning.dm

```diff
@@ -392,6 +395,14 @@
 		SSnanoui.update_uis(src)
 		return
 
+	for(var/obj/machinery/clonepod/pod in pods)
+		if(pod.occupant && pod.occupant.ckey == null)
+			for(var/datum/soullink/soulhook/hook in pod.sharedSoulhooks) //Check the soul hooks to see if the to be cloned person is in here
+				if(hook.soulowner.ckey == subject.ckey)
```

Comment:
I wouldn't rely on `soulowner` - the hook is "attached" to the originally scanned body, such that if it is revived somehow, the cloner will malfunction and destroy the no longer needed clone. The soulhook is not placed here with the intent to be checked like this.

Instead, use `pod.clonemind`, which is the `mind` datum that will be placed in the clone upon completion. You should be able to do `pod.clonemind == subject.mind`, since only a single instance of a mind datum exists per "character", and this property is integral to proper function of a clone pod. (otherwise, they wouldn't be placed in the clone when it completes).